packages/components/src/DataList/DataList.module.css

```diff
@@ -26,10 +26,9 @@
 }
 
 .headerFilters {
-  display: grid;
```

Comment:
I'm open to grid still being a potential way to handle this layout, but I'm not certain what we'd actually gain from it

Also, looking at this whole selector, I'm wondering if I can't just use `<Flex>` here 🤔

Reply:
The use of `:empty` just below is a nice touch but it feels like that could also be handled by not rendering the element at all rather than rendering and then checking its' contents 🤷